space-cli/cmd/modules/ingress/delete.go

```diff
@@ -0,0 +1,52 @@
+package ingress
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/filter"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/transport"
+)
+
+func deleteIngressGlobalConfig(project string) error {
+
+	// Delete the ingress global config from the server
+	url := fmt.Sprintf("/v1/config/projects/%s/routing/ingress/global", project)
+
+	payload := new(model.Response)
+	if err := transport.Client.MakeHTTPRequest(http.MethodPost, url, map[string]string{}, payload); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func deleteIngressRoute(project, prefix string) error {
+
+	objs, err := GetIngressRoutes(project, "ingress-route", map[string]string{}, []string{})
+	if err != nil {
+		return err
+	}
+
+	doesRouteExist := false
```

Comment:
What is the use of this boolean?

Reply:
Used in differentiating when to survey only filtered resource IDs and when to show all

Reply:
You don't need to pass this as a parameter to the function. Just create the boolean locally inside the Deleteoptions function.

The boolean that you are passing as a parameter is always false